### DISCUSSION TITLE
URL Linking?

### DISCUSSION THREAD

**Discussion by chriscrofty** [joelekstrom/fastmate#78] [Ideas]
Is it possible to add URL linking such as fastmate:// .... to link to email items? I am using Things 3 which can autofill a task on quick entry with a link from another application. This would save me a tonne of time and allow me to connect it to other apps.

Things 3 website support page: https://culturedcode.com/things/support/articles/2249437/

_The app needs to have an AppleScript interface so that Autofill can access a selection within it. Specifically, the developer needs to support linking to objects in the app via URLs: for example, Mail uses message:// and Safari uses http:// or https://._

**Comment by joelekstrom** [comment]
Hello @thedaddy123! And sorry for the slow reply. Yes, this is already supported. Within Fastmate, click Edit -> Copy Fastmate URL to get an app URL to the current mail or label. It will be something like `fastmate://app/mail/Inbox/`

I hope that helps!
